access.md

```diff
@@ -15,32 +15,15 @@ The serial console of a virtual machine can be accessed by using the `console`
 command:
 
 ```bash
-$ virtctl console -s http://mycluster:8184 testvm
+$ virtctl console --kubeconfig=$KUBECONFIG testvm
 ```
 
-### Accessing the graphical console
+### Accessing the graphical console (VNC)
 
 Accessing the graphical console of a virtual machine is usually done through
-SPICE, which requires a SPICE client like `remote-viewer`. Once the tool is
+VNC, which requires a VNC client like `remote-viewer`. Once the tool is
```

Comment:
Now that I'm reading this, it makes it seem like remote-viewer is just one of many VNC clients we support when in reality it's the only one virtctl works with right now.

Reply:
Fixed.